wordpress-mobile/WordPress-Android
# Feature: feedback form v1

This PR is the feature branch for the feedback form and contains the following PRs:

* #21072
* #21073
* #21081
* #21094

To test:

* Switch to the Me tab
* Tap "Send feedback"
* Note that the Submit button on the feedback form isn't enabled until you start typing
* Tap the close icon after entering text and confirm you're asked whether to discard changes
* Tap the close icon without entering text and confirm no discard dialog is shown
* Enter text and tap the Submit button
* Verify the feedback is submitted
* You'll receive an email which includes the ticket number (**Aside:** should this email be sent?). Verify the ticket contains the entered text by visiting `https://a8c.zendesk.com/agent/tickets/[TICKET-NUMBER]` then mark the ticket as resolved (note that you may need to request access in the [Zendesk Account Manager](https://mc.a8c.com/happiness/zendesk-account-manager/?instance=automattic)). 
* Turn on airplane mode and try to submit another message, then verify you're told there's no connection
* Reset your Zendesk identity by clearing "supportEmail" and "supportName" in `AppPrefs`, then verify you can still submit feedback (you might want to set a breakpoint on `ZendeskHelper.createAnonymousIdentityIfNeeded` to verify an anonymous identity is used)

https://github.com/user-attachments/assets/427672cc-1c13-4ed6-8b6f-f40225098b1f
